src/NHSD.BuyingCatalogue.Documents.API/Startup.cs

```diff
@@ -32,6 +32,8 @@ public void ConfigureServices(IServiceCollection services)
                     .GetBlobContainerClient(settings.ContainerName);
             });
 
+            services.AddHealthChecks().AddAzureBlobStorage(settings.ConnectionString, settings.ContainerName);
```

Comment:
In the Buying Catalogue API I quite like the use of an extension method here so we can group the custom health checks for the API.